pkg/utils/parse.go

```diff
@@ -198,3 +212,40 @@ func ByteCountIEC(b int64) string {
 	return fmt.Sprintf("%.1f %ciB",
 		float64(b)/float64(div), "KMGTPE"[exp])
 }
+
+// ParseGitlab will parse gitlab config from address
+func ParseGitlab(addr, repo string) (string, *Content, error) {
+	// We support two valid format:
+	// 1. https://example.gitlab.com/<owner>/<repo>
+	// 2. https://example.gitlab.com/<owner>/<repo>/tree/<branch>
+	URL, err := url.Parse(addr)
+	if err != nil {
+		return "", nil, err
+	}
+
+	arr := strings.Split(addr, repo)
+	owner := strings.Split(arr[0], URL.Host+"/")
```

Comment:
It's better to make sure that the length of arr is greater than 1 here.

Reply:
As long as split succeeds, the arr will be greater than 1